### PR TITLE
Group page

### PR DESCRIPTION
**Pull request by paradime**
fixes https://github.com/PagerNation/PagerNation/issues/113

### CODE REVIEW

File: src/routes/Group/components/GroupView.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./GroupView.scss";
```
Comment (jarrydlee) [inline src/routes/Group/components/GroupView.js]: Just curious, how does that get inserted into the component?

Comment (kwm4385) [inline src/routes/Group/components/GroupView.js]: Webpack magic

Comment (jarrydlee) [inline src/routes/Group/components/GroupView.js]: Say no more

File: src/routes/Group/components/GroupView.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class GroupView extends React.Component {
+
+  componentWillMount() {
+    this.props.fetchGroup(this.props.params.groupId)
+  }
+
+  render() {
+    console.log(this.props.group);
```
Comment (jarrydlee) [inline src/routes/Group/components/GroupView.js]: Do we need this?

**Comment by paradime** [comment]
image

**Comment by paradime** [comment]
@kwm4385 I think this is good as is, we can itterate once the backend team populates user in the routes. Do you have any feelings about the UI?

**Comment by kwm4385** [comment]
I agree we'll need to wireframe this out and discuss more. Let talk during the meeting tomorrow.

**Comment by paradime** [comment]
image

**Comment by paradime** [comment]
Wireframe: 
image

**Comment by kwm4385** [comment]
Can you merge master into this branch?

**Comment by kalebdavis** [comment]
Rebase* master into the branch

**Pull request merged**